NEWS.md

```diff
@@ -1,3 +1,12 @@
+1.0.0-rc1 (2020-06-19)
+======================
+
+Plugins
+-------
+
+- Add [Redis Queue (RQ)](https://python-rq.org/) Launcher plugin ([Jan-Matthis](https://github.com/jan-matthis))
```

Comment:
ah, no.
read [news entries](https://hydra.cc/docs/development/contributing#news-entries) in the developer guide.

Reply:
Ah! It's fixed now